source/clear-linux/guides/maintenance/mixin.rst

```diff
@@ -1,101 +1,116 @@
 .. _mixin:
 
-Create and add custom bundles to your upstream Clear Linux system
-#################################################################
+Mixin tool
+##########
 
-|CLOSIA| offers many curated bundles that you can install on your system to
-create your desired capabilities. If the available upstream bundles do not
-meet your needs, you can create and add your own custom bundles to your
-system using one of two methods. Note: Upstream refers to the official
-version of |CL|.
+The mixin tool, a light wrapper for :ref:`mixer`, is used to customize your
+installation of |CL-ATTR| by *side loading* a custom bundle. This is known
+as a *mixin*. Creating a mixin is useful when you need to add custom or 3rd
+party content but want to keep on the upstream update cycle. This guide covers
+the workflow of creating your own mixin using the mixin tool.
 
```

Comment:
Great revision of  intro  for `mixin`. This nicely clarifies the general difference between them.